lib/util/napi.js

```diff
@@ -166,16 +166,14 @@ module.exports.get_napi_build_version_from_command_args = function(command_args)
 
 module.exports.swap_build_dir_out = function(napi_build_version) {
   if (napi_build_version) {
-    const rm = require('rimraf');
-    rm.sync(module.exports.get_build_dir(napi_build_version));
+    fs.rm(module.exports.get_build_dir(napi_build_version), { recursive: true, force: true });
```

Comment:
Apologies if you interpreted this way (I only meant to say don't use rmdirSync as opposed to rmSync, not to switch to the async version too), but by switching this to the asynchronous version the behaviour here has slightly changed. Change back to `fs.rmSync`, if it's not too much bother.